src/python/turicreate/test/test_modules.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2019 Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can
+# be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+from __future__ import print_function as _
+from __future__ import division as _
+from __future__ import absolute_import as _
+import unittest
+import turicreate as tc
+import inspect
+
+class LoadModuleTest(unittest.TestCase):
```

Comment:
I think we have a test_api_visilibiy. You should add this as an additional test there or make that test work :)

Reply:
K!